modules/tinymce/src/core/main/ts/api/commands/IndentCommands.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import * as IndentOutdent from '../../commands/IndentOutdent';
+import type Editor from '../Editor';
+
+export const registerCommands = (editor: Editor) => {
+  editor.editorCommands.addCommands({
+    Indent: () => {
+      IndentOutdent.indent(editor);
+    },
+
+    Outdent: () => {
+      IndentOutdent.outdent(editor);
+    },
+  });
+
+  editor.editorCommands.addCommands({
```

Comment:
Should this be broken up into separate functions for exec and query commands? That's been done everywhere else and it wasn't immediately obvious why this was separate (though it only took reading until the end of the function call to figure it out).